### DISCUSSION TITLE
Release 1.2.0 Beta 2

### DISCUSSION THREAD

**Discussion by apohl79** [apohl79/audiogridder#510] [General]
## PLUGIN
- Added an option for audio/MIDI data transfer on play/rec only
	- Reverts back to 1.1 behaviour where audio and MIDI data gets transferred always
	- Resolves the issue of instruments only playing when the DAW is in replay/record mode
- Resolved “DAW hanging” issue on Windows when the tray app is not installed correctly
	- Added error message to inform the user to fix the installation
- Fixed Windows installer where the tray app was not installed correctly when upgrading from 1.1
- Allow MIDI only instruments
- Fixed AudioStreamer issue that could crash the plugin/DAW
- Filter presets to match creation mode (FX, instrument, MIDI)

## SERVER
- Simple idling of the screen worker for no capturing instead of running the capturing routine

<hr /><em>This discussion was created from the release <a href='https://github.com/apohl79/audiogridder/releases/tag/release_1_2_beta2'>Release 1.2.0 Beta 2</a>.</em>

**Comment by dhstsw** [comment]
HI,

I must say that this release has ways better performances on my setup (server running on a VM under KVM, Win10 64).
Clicks, pops and general audio glitches are nearly gone.

Tho, the automation issue (as better described in the link below) is still there.

https://github.com/apohl79/audiogridder/discussions/485

If  i may add a suggestion: add a "don't lose focus" option. Whenever (for any reason) another program gets focus on windows, AG server become unresponsive (until it gets focus again).


Thanks for the amazing work!

**Reply by abysalvinthomas** [reply]
Restarting server around 3 times in a span of 5 minutes... so i installed back 1.1 stable release

**Reply by apohl79** [reply]
> Restarting server around 3 times in a span of 5 minutes... so i installed back 1.1 stable release

would you consider filing a real bug report as this info does not provide any help solving the problem? https://audiogridder.com/bug-reports/

thx!

**Comment by apohl79** [comment]
> Tho, the automation issue (as better described in the link below) is still there.
> 
> https://github.com/apohl79/audiogridder/discussions/485

Ok i see, i will have to figure out what live's logic is here.

> If  i may add a suggestion: add a "don't lose focus" option. Whenever (for any reason) another program gets focus on windows, AG server become unresponsive (until it gets focus again).

If ag opens/reopens a plugin ui, it should bring the plugin window to front. does this not work for you if you reopen the ag plugin in the daw for example?

> Thanks for the amazing work!

thanks :-)

**Comment by LondonAcoustics** [comment]
Audiogridder plugin latency:

- not auto compensated in Cubase Pro 10.5 (it goes out of sync with the oter non-audiogridderised tracks, basically)
- auto compensated in Studio One 5 only if clicked once in the latency list (different value or the same as default)
- auto compensated fully in Reper 6

Tested at 48khz/24 bit, Windows 10 Pro (both PCs), Audiogridder Beta 1.2.0 Beta2, VST2 format

**Comment by fojerhar** [comment]
Hi
I have 3 computers do test it. Same pt versions 2021.3.0. Used the same plugins in AG.
1. at work, 2012 iMac with High Sierra. AG 1.2.0.b2 works perfectly!
2. laptop 2018 MBP catalina. 1.2.0.b2 i can open plugins, work with it durin playback, as I stop playback PT quits
[MBP AG 1.2.0b2.zip](https://github.com/apohl79/audiogridder/files/6369399/MBP.AG.1.2.0b2.zip)
3. laptop 2018 MBP catalina 1.1.1 works fine
[MBP AG 1.1.1.zip](https://github.com/apohl79/audiogridder/files/6369401/MBP.AG.1.1.1.zip)
4. AMD hackintosh catalina, 1.2.0.b2 Same resutls as MBP
5. Amd hack with 1.1.1:
[Hack AG 1.1.1.zip](https://github.com/apohl79/audiogridder/files/6369419/Hack.AG.1.1.1.zip)

Strange issue since a week (since 1.2.0.b1 came out): at hack I cannot open a new plug instance in a new session.
If I open one what I made a week before, it opens and runs fine. If I open a new ag in this session pt quits (with 1.1.1 also) when I stop playback. Even with not any plugins in it. Open okay, play okay, stop playack PT quits.
[Hack AG 1.1.1 old session.zip](https://github.com/apohl79/audiogridder/files/6369409/Hack.AG.1.1.1.old.session.zip)

Tried to delete all instances of 1.2.0.b2, plists, everithing I have found (honestly what forklift found)
If I try to install 1.1.1 after 1.2.0.b2 installer do not change the aax plugins, 1.2.0 version stays, have to delete if by hand before reinstalling 1.1.1

**Comment by Pabloscotty** [comment]
Hey in Studio One 5.2 Mac - Sometimes plugin window opens in very small size. Reopening it restores it but closing and reopening again - make it small again.

**Comment by CaBuffalo** [comment]
Hi, Pabloscotty,

I saw this issue in S1 before. It never happens in Logic. I'm pretty sure
it's the compatibility from S1. Let's see what Andreas say.

On Sat, Apr 24, 2021 at 4:32 AM Pabloscotty ***@***.***>
wrote:

> Hey in Studio One 5.2 Mac - Sometimes plugin window opens in very small
> size. Reopening it restores it but closing and reopening again - make it
> small again.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/510#discussioncomment-653286>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6HYFZL6QM2AQ6B2NBDTKKT5LANCNFSM43LHMTBA>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by Pabloscotty** [reply]
Logic works with AU but here in S1 in I'm using VST2 version I'll check the AU in S1.

**Reply by Pabloscotty** [reply]
The VST2 and AudioUnits - works prefect. The problem only exist with VST3

**Comment by CaBuffalo** [comment]
Hi, Viktor Szabo and LondonAcustics:

Thanks for your feedback. Please stay tuned.

On Sat, Apr 24, 2021 at 3:42 AM Viktor Szabó ***@***.***>
wrote:

> Hi
> I have 3 computers do test it. Same pt versions 2021.3.0. Used the same
> plugins in AG.
>
>    1. at work, 2012 iMac with High Sierra. AG 1.2.0.b2 works perfectly!
>    2. laptop 2018 MBP catalina. 1.2.0.b2 i can open plugins, work with it
>    durin playback, as I stop playback PT quits
>    MBP AG 1.2.0b2.zip
>    <https://github.com/apohl79/audiogridder/files/6369399/MBP.AG.1.2.0b2.zip>
>    3. laptop 2018 MBP catalina 1.1.1 works fine
>    MBP AG 1.1.1.zip
>    <https://github.com/apohl79/audiogridder/files/6369401/MBP.AG.1.1.1.zip>
>    4. AMD hackintosh catalina, 1.2.0.b2 Same resutls as MBP
>    5. Amd hack with 1.1.1:
>    Hack AG 1.1.1.zip
>    <https://github.com/apohl79/audiogridder/files/6369419/Hack.AG.1.1.1.zip>
>
> Strange issue since a week (since 1.2.0.b1 came out): at hack I cannot
> open a new plug instance in a new session.
> If I open one what I made a week before, it opens and runs fine. If I open
> a new ag in this session pt quits (with 1.1.1 also) when I stop playback.
> Even with not any plugins in it. Open okay, play okay, stop playack PT
> quits.
> Hack AG 1.1.1 old session.zip
> <https://github.com/apohl79/audiogridder/files/6369409/Hack.AG.1.1.1.old.session.zip>
>
> Tried to delete all instances of 1.2.0.b2, plists, everithing I have found
> (honestly what forklift found)
> If I try to install 1.1.1 after 1.2.0.b2 installer do not change the aax
> plugins, 1.2.0 version stays, have to delete if by hand before reinstalling
> 1.1.1
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/510#discussioncomment-653201>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6HUCZP5R6FPNTZAH7DTKKOAVANCNFSM43LHMTBA>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by fojerhar** [reply]
HI
Just two questions: can you see well the anomalies happening trough the trace files, or just can see that something is not proper.

Q2: at my iMac with high sierra the 1 block size is 10ms, at my other computer 1 block is 21ms. Both sessions are at 48K. Does this affect the buffering and latency?

**Reply by apohl79** [reply]
> can you see well the anomalies happening trough the trace files, or just can see that something is not proper.

misbehavior is hopefully visible via log files. i use trace files 
mostly to help analyze crashes in combination with core dumps.

> at my iMac with high sierra the 1 block size is 10ms, at my other computer 1 block is 21ms. Both sessions are at 48K. Does this affect the buffering and latency?

the i/o buffer setting in your daw defines the size of one block. a block in ag is one full daw i/o buffer. the larger it is, the higher the latency.
